automation/setup.go

```diff
@@ -141,8 +168,7 @@ func ProvisioningTestTeardown(dbSecret, ndbSecret *corev1.Secret, database *ndbv
 
 	// Delete Database
 	if database != nil {
-		database.Spec.NDB.Server = os.Getenv("NDB-SERVER")
-		database.Spec.NDB.ClusterId = os.Getenv("NDB-CLUSTER-ID")
+		database.Spec.Instance.ClusterId = os.Getenv("NDB-CLUSTER-ID")
```

Comment:
Setting this env variable only specific for the automation tests?

Reply:
Yes.